### ISSUE TITLE
A question about  the "INFOR" field

### ISSUE THREAD

**Issue by rain-zjg** [xunchen85/ERVcaller#26]
Hello, Dr. Xun Chen,
I have a question about the INFO column. For the "INFOR" field, the description stands for "NAME,START,END,LEN,DIRECTION,STATUS". However, it seems that the "START" is not always 1, here is some examples of my results:

#CHROM  POS     ID      REF     ALT     QUAL    FILTER  INFO 
chr01   629412  .       T       <INS_MEI:TE_00011020_LTR#ClassII_DNA_Mutator_nMITE>     .       PASS    CR=114;GR=0,0.656,0.745,0.633,0.659,0.444,0.333;GTF=YES;INFOR=TE_00011020_LTR#DNA/Mutator,571,839,269,+,5;SR=8
chr01   951226  .       G       <INS_MEI:TE_00005384#ClassII_DNA_hAT_MITE>      .       PASS    CR=23;GR=0,0.581,0.359;GTF=YES;INFOR=TE_00005384#DNA/hAT,3526,6274,2749,+,5;SR=9
chr01   951675  .       T       <INS_MEI:TE_00003559_INT#ClassI_LTR_Copia>      .       PASS    CR=5;GR=0,0.556,0.467;GTF=YES;INFOR=TE_00003559_INT#LTR/Copia,1836,1908,73,-,5;SR=7
chr01   986317  .       A       <INS_MEI:TE_00006433_LTR#ClassI_LTR_Gypsy>      .       PASS    CR=9;GR=0,1,0.444,0.5;GTF=YES;INFOR=TE_00006433_LTR#LTR/Gypsy,6,2539,2534,+,5;SR=6
chr01   990068  .       A       <INS_MEI:TE_00011590_LTR#ClassI_LTR_Gypsy>      .       PASS    CR=8;GR=0,0.611;GTF=YES;INFOR=TE_00011590_LTR#LTR/Gypsy,2024,2437,414,+,5;SR=3

I'm wondering what's the insertion range of a TE insertion. For example, in the first record, the POS is chr01:629412, while the "START" of "INFOR" field is 571, does that mean the actual start position for the insertion is "629412 + 571" and the end position is "629412 + 839" ? Or the start position is 629412? In my current study, I need to obtain the start and end postion of TE insertions. I'm confused and could you do me a favor? 
Best wishes,
Rain

**Comment by xunchen85** [comment]
Hi Rain,

The "Start" and "End" are the positions of the inserted TE fragments. The TE insertion is not full length usually. 

For example, the first example means that 571-839 bps of the LTR reference sequence was inserted, and the total inserted length was 269 bp. the insertion direction is "forward". 

Thanks,
Xun

**Comment by rain-zjg** [comment]
Thank you very much, your answer is great. However, I have another question: what's  the start and end coordinates on the genome for a TE insetion? I have checked all the issues on your homepage, and as you mentioned before, the "CHROM" and "POS" columns indictae the breakpoint postion, I wonder if the coordinates are "CHROM:POS ± the length of a TE insertion" and strand should be considered to determine which side is start or end. Here are some examples of TE insertion coordinates for Arabdopsis thaliana:
Chr	start	stop	superfamily	TE	pres/abs	
1	8667	8700	MuDR	ATDNAI27T9A	presence
1	10585	10600	Gypsy	ATHILA6A	presence
1	15091	15099	Copia	ATCOPIA13	presence
1	15894	15898	Gypsy	ATGP1	presence	
1	20514	20532	MuDR	VANDAL8	presence	
1	20530	20537	Gypsy	ATGP1	presence	
All I need are "CHROM",  "START coordinates" and  "END coordinates". Could you explain it using the data of my ERVcaller results?

**Comment by xunchen85** [comment]
Indeed, in our final vcf output file, we only report one integration breakpoint. We assume the TE insertion will integrate into one location (nucleotide). The VCF format could also be easily used for other tools. 

Of course, as you showed, they may also lead to some insertions/deletions, thus in some cases, there will be two breakpoints or a range. ERVcaller will report the most precise position in the VCF output. Maybe you could check if you could use "TSD" instead for some loci, which refers to the tandem site duplication. In this case, you could do coordinate + TSD length to get a range.

Best,
Xun

**Comment by rain-zjg** [comment]
Thank you most sincerely. That's a good suggestion!
